test/extensions/filters/http/common/fuzz/filter_fuzz_test.cc

```diff
@@ -58,6 +61,12 @@ DEFINE_PROTO_FUZZER(const test::extensions::filters::http::FilterFuzzTestCase& i
   try {
     // Catch invalid header characters.
     TestUtility::validate(input);
+    // Somehow (probably before the mutator was extended by a filter) the fuzzer picked up the
+    // extensions it can not fuzz (like the ext_* ones). Therefore reject these here.
+    if (exclusion_list.count(input.config().name()) != 0) {
```

Comment:
nit prefer `contains()` as it's more readable

Reply:
aah, I should have checked properly first! :-(

Reply:
done